cypher-shell/src/main/java/org/neo4j/shell/prettyprint/PrettyPrinter.java

```diff
@@ -121,14 +118,14 @@ private String mapAsString(Map<String, Object> map) {
         StringBuilder sb = new StringBuilder("{");
         sb.append(
                 map.entrySet().stream()
-                        .map(e -> escapeIfNeeded(e.getKey()) + COLON_SEPARATOR + e.getValue())
+                        .map(e -> cypherVariablesFormatter.escape(e.getKey()) + COLON_SEPARATOR + e.getValue())
```

Comment:
:) I see it now. It makes it immutable as well I suppose. Oh well, I am swayed :) I will update the PR